### PR TITLE
Chore: Add 1.1.0 Schema to PR Template

### PR DESCRIPTION
**Pull request by Trenly**
With the pipelines now (mostly) allowing for 1.1.0 schemas, the PR Template should be updated to reflect this

@denelon

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/44282)

### CODE REVIEW

**Review by jedieaston — APPROVED** [review]

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 - [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
 - [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
 - [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
-- [ ] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?
```
Comment (jedieaston) [inline .github/PULL_REQUEST_TEMPLATE.md:5]: I still think this template should be updated, but maybe the Manifest Spec doc should be too? Maybe if there's a blog post about the schema changes somewhere that would be good to link to, but linking to the JSON schemas is less clear IMO.

**Comment by denelon** [comment]
https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#what-next will need an update as well. I need to comb the other markdown documents to make sure they reflect 1.1 properly as well.

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (denelon) [reply .github/PULL_REQUEST_TEMPLATE.md:5]: I haven't written the blog for the 1.2 client including the new schema additions. More documentation to write :)

Comment (Trenly) [reply .github/PULL_REQUEST_TEMPLATE.md:5]: > I still think this template should be updated, but maybe the Manifest Spec doc should be too? Maybe if there's a blog post about the schema changes somewhere that would be good to link to, but linking to the JSON schemas is less clear IMO.

It looks like the manifest spec doc does need to be updated. It looks like all the same information is over at https://docs.microsoft.com/en-us/windows/package-manager/package/manifest  and is contained in multiple markdown files over at the cli repository.

I noticed too that the official docs site says that manifest specification has `a complete list and descriptions of items in a manifest` when it doesn't. So, I started to remove the duplicate information and actually create the list and descriptions of items in a manifest. Take a look and provide feedback - https://github.com/microsoft/winget-cli/compare/master...Trenly:Docs110

If you think it's worth continuing on, I'll make a new docs issue over at CLI and have denelon assign it to me

Comment (denelon) [reply .github/PULL_REQUEST_TEMPLATE.md:5]: We've been having challenges keeping documentation in sync at Microsoft Docs and at GitHub. We need a better system. Microsoft Docs has some additional special formatting that's hard to replicate and keep correct here. We may need to cover the basics at Microsoft Docs and refer to GitHub for the "latest" updates.

**Comment by denelon** [comment]
https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.0.0 is where I was documenting all the schema elements and attempting to keep them valid with respect to the client implementation.

**Comment by Trenly** [comment]
I'm going to close this, given the discussions that have been happening in Gitter Chat

**Pull request closed without merging**